### PR TITLE
Make signedness of nBytes consistent

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/coreclr/src/vm/i386/stublinkerx86.cpp [hunk 1]
```diff
@@ -3845,7 +3845,7 @@ VOID StubLinkerCPU::EmitShuffleThunk(ShuffleEntry *pShuffleEntryArray)
 
     UINT16 emptySpot = 0x4 | ShuffleEntry::REGMASK;
 
-    while (true)
+    for (;;)
```
Comment (am11) [inline src/coreclr/src/vm/i386/stublinkerx86.cpp:3848]: Why was this change necessary? `while (true)` is preferred over `for (;;)` because it is clearer and avoids unnecessary indentation changes (`for (; ;)` vs. `for (; ; )`).

**Comment by jkotas** [comment]
https://github.com/dotnet/runtime/pull/44130#issuecomment-720209158

**Comment by pi1024e** [comment]
I should have made myself more clear; this is to fix an unsignedness warning by the compiler, because every other variable is supposed to be unsigned, from the argument, to the value it is being assigned. In this particular case, the rest of the code expects it to be unsigned.

This is not like the last PR.

**Comment by jkotas** [comment]
What is the compiler warning that this is fixing? What is the compiler version that you are getting it with?

**Comment by pi1024e** [comment]
That would be MSVC, the one that the tests are using. I removed the other changes that are unrelated to the warning.

**Comment by jkotas** [comment]
> That would be MSVC, the one that the tests are using. 

We build with warnings-as-errors and we are not seeing this warning.

**Comment by ViktorHofer** [comment]
// Auto-generated message

https://github.com/dotnet/runtime/commit/69e114c1abf91241a0eeecf1ecceab4711b8aa62 which was merged 12/7 removed the intermediate src/coreclr/**src**/ folder. This PR needs to be updated as it touches files in that directory which causes conflicts.

To update your commits you can use this bash script: https://gist.github.com/ViktorHofer/6d24f62abdcddb518b4966ead5ef3783. Feel free to use the comment section of the gist to improve the script for others.

**Comment by jkotas** [comment]
Closing stale PRs.
